libtwolame/mem.c

```diff
@@ -35,13 +35,10 @@
 
 void *twolame_malloc(size_t size, int line, char *file)
 {
-    void *ptr = (void *) malloc(size);
+    void *ptr = (void *) calloc(size, 1);
 
-    if (ptr != NULL) {
-        memset(ptr, 0, size);
-    } else {
+    if (ptr == NULL) {
         fprintf(stderr, "Unable to allocate %d bytes at line %d of %s\n", (int) size, line, file);
-        return NULL;
```

Comment:
Nothing but readability.
In fact, the same "return NULL" would be performed anyway by the terminating "return (ptr)".